sdk/avs/Azure.ResourceManager.Avs/src/Customized/ArmAvsModelFactory.cs

```diff
@@ -45,13 +46,236 @@ public static partial class ArmAvsModelFactory
         /// <param name="nsxPublicIPQuotaRaised"> Flag to indicate whether the private cloud has the quota for provisioned NSX Public IP count raised from 64 to 1024. </param>
         /// <returns> A new <see cref="Avs.AvsPrivateCloudData"/> instance for mocking. </returns>
-        public static AvsPrivateCloudData AvsPrivateCloudData(ResourceIdentifier id = null, string name = null, ResourceType resourceType = default, SystemData systemData = null, IDictionary<string, string> tags = null, AzureLocation location = default, string skuName = null, ManagedServiceIdentity identity = null, AvsManagementCluster managementCluster = null, InternetConnectivityState? internet = null, IEnumerable<SingleSignOnIdentitySource> identitySources = null, PrivateCloudAvailabilityProperties availability = null, CustomerManagedEncryption encryption = null, AvsPrivateCloudProvisioningState? provisioningState = null, ExpressRouteCircuit circuit = null, AvsPrivateCloudEndpoints endpoints = null, string networkBlock = null, string managementNetwork = null, string provisioningNetwork = null, string vMotionNetwork = null, string vCenterPassword = null, string nsxtPassword = null, string vCenterCertificateThumbprint = null, string nsxtCertificateThumbprint = null, IEnumerable<ResourceIdentifier> externalCloudLinks = null, ExpressRouteCircuit secondaryCircuit = null, NsxPublicIPQuotaRaisedEnum? nsxPublicIPQuotaRaised = null)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static AvsPrivateCloudData AvsPrivateCloudData(
+            ResourceIdentifier id = null,
+            string name = null,
+            ResourceType resourceType = default,
+            ResourceManager.Models.SystemData systemData = null,
+            IDictionary<string, string> tags = null,
+            AzureLocation location = default,
+            string skuName = null,
+            ManagedServiceIdentity identity = null,
+            AvsManagementCluster managementCluster = null,
+            InternetConnectivityState? internet = null,
+            IEnumerable<SingleSignOnIdentitySource> identitySources = null,
+            PrivateCloudAvailabilityProperties availability = null,
+            CustomerManagedEncryption encryption = null,
+            AvsPrivateCloudProvisioningState? provisioningState = null,
+            ExpressRouteCircuit circuit = null,
+            AvsPrivateCloudEndpoints endpoints = null,
+            string networkBlock = null,
+            string managementNetwork = null,
+            string provisioningNetwork = null,
+            string vMotionNetwork = null,
+            string vCenterPassword = null,
+            string nsxtPassword = null,
+            string vCenterCertificateThumbprint = null,
+            string nsxtCertificateThumbprint = null,
+            IEnumerable<ResourceIdentifier> externalCloudLinks = null,
+            ExpressRouteCircuit secondaryCircuit = null,
+            NsxPublicIPQuotaRaisedEnum? nsxPublicIPQuotaRaised = null)
```

Comment:
we should change those shared parameter with the generated version of this method to required.